examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -502,10 +508,25 @@ void WindowManager::Cover::CallbackPositionSet(intptr_t arg)
     position.SetNonNull(data->percent100ths);
 
     if (data->isTilt)
+    {
         TiltPositionSet(data->mEndpointId, position);
+#ifdef DIC_ENABLE
+        uint16_t value = data->percent100ths;
+        char buffer[MSG_SIZE];
+        itoa(value, buffer, DECIMAL);
```

Comment:
So if value is maximal (10000), and we try to print it to a 5-char buffer... where does the null terminator go?